pkg/sfu/streamallocator.go

```diff
@@ -1103,9 +1229,13 @@ func (t *Track) WritePaddingRTP(bytesToSend int) int {
 	return t.downTrack.WritePaddingRTP(bytesToSend)
 }
 
-func (t *Track) AdjustAllocation(availableChannelCapacity uint64) (uint64, uint64) {
-	t.bandwidthRequested, t.optimalBandwidthNeeded = t.downTrack.AdjustAllocation(availableChannelCapacity)
-	return t.bandwidthRequested, t.optimalBandwidthNeeded
+func (t *Track) AdjustAllocation(availableChannelCapacity uint64) (bool, bool, uint64, uint64) {
```

Comment:
maybe have a named return values is better, now we have 4 return value, have to read code to verify the mean of return value. like:
`func(t *Track) AdjustAllocation(....)(isPausing,isResuming bool, bandwidthRequested, optimalBandwidthNeeded uint64){}`

Reply:
Good idea @cnderrauber . Will do.

Reply:
Done in this commit - https://github.com/livekit/livekit-server/pull/155/commits/36b231d44c9c45c3dcbfe3536dacb71596850364